docs/_articles/en/release-information.md

```diff
@@ -6,6 +6,17 @@ lang: en
 ### To update the plugin, please follow the linked [instructions](./en/getting-started/install/#upgrade-plug-in).
 <br>
 
+## [v2.13.9](https://github.com/forcedotcom/sfdx-scanner/tree/v2.13.9) (09-14-2022)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.13.8...v2.13.9)
+
+### Release Summary
+* NEW: Updates to local RetireJS Vulnerability Repository.
```

Comment:
@jfeingold35 Per the release notes Quip (https://salesforce.quip.com/EKksAvKaHPbJ) this should be:

* NEW: We made some updates to the RetireJS vulnerability database.